tests/unit/dataactvalidator/test_a1_appropriations.py

```diff
@@ -56,4 +56,4 @@ def test_failure(database):
 
     # Non-overlapping ranges of agency IDs should generate two errors
     errors = number_of_errors(_FILE, database, models=[tas, tas_null, approp, approp_null])
-    assert errors == 2
\ No newline at end of file
+    assert errors == 2
```

Comment:
I guess these are files that don't end with a blank line, put a question in the summary about that.